build/props/common.props

```diff
@@ -24,13 +24,13 @@
         </Description>
     </PropertyGroup>
     <!-- SourceLink -->
-    <PropertyGroup>
+    <PropertyGroup Condition="'$(EnableSourceLink)' == ''">
```

Comment:
I think this may conflcit with an existing (or proposed) SDK property, so we should probably prefix it with Silk.